build/squire-raw.js

```diff
@@ -2254,17 +2254,33 @@ var onDrop = function ( event ) {
     while ( l-- ) {
         switch ( types[l] ) {
         case 'text/plain':
+        case 'Text': // IE Specific
             hasPlain = true;
             break;
         case 'text/html':
             hasHTML = true;
             break;
         default:
-            return;
+            break;
         }
     }
+
     if ( hasHTML || hasPlain ) {
+        var range = this.getSelection();
         this.saveUndoState();
+        this.setSelection( range );
```

Comment:
Why are you restoring the selection to where it was before the drag? That seems wrong.